vllm/multimodal/base.py

```diff
@@ -43,9 +43,6 @@ def try_concat(
         *,
         device: torch.types.Device,
     ) -> BatchedTensors:
-        # Avoid initializing CUDA too early
-        import torch
-
```

Comment:
Was this issue resolved?

Reply:
Yeah, it's resolved. It has nothing to do with the `torch` import itself (and you can see that I import it at the top of the module already, so this one is redundant)